docs/content/interop.md

```diff
@@ -25,7 +25,7 @@ You need to use the ICG to generate `.schema` files and type bindings for any ne
 ### How to use the ICG
 As the Interop Code Generator is a plugin for the Unreal toolbar, to use it you must first:
 *  build your game project to access the SpatialOS fork of Unreal Engine as well as the Unreal GDK. (You do this as part of the [installation and setup](../setup-and-installing.md), swapping the “Starter Project” and its file location for your game’s name and file location.)
-* list the classes that you want to create typebindings for in the in the **SpatialOS Unreal GDK** settings in the Unreal Engine 4 Editor. For instructions on how to do this, see the guidance on [Setting up the Interop Code Generator](#Setting-up-the-Interop-Code-Generator), below.
+* tag the classes that you want to create typebindings for. For instructions on how to do this, see the guidance on [Setting up the Interop Code Generator](#eetting-up-the-interop-code-generator), below.
 
```

Comment:
Thanks for fixing the dodgy link. Suspect new and improved link has a typo: `#eetting` - `#setting`.

Reply:
I tried...